portfolio/src/components/Section.tsx

```diff
@@ -0,0 +1,16 @@
+import clsx from "clsx";
+import type { FC, HTMLAttributes } from "react";
+
```

Comment:
Sorry to keep repeating myself... Another custom component because of the same `className` being applied a bunch of places. Just a nice abstraction to have as well in case you want to add a new section to the home page.